src/group/views.py

```diff
@@ -6,13 +6,15 @@
 
 
 def index(request):
-    id_group = request.GET.get('id')
 
+    id_group = request.GET.get('id')
     if id_group is not None:
         if not id_group.isdigit():
             return HttpResponse("'id' must be numeric and greatest when zero")
 
         id_group = int(id_group)
+        if id_group == 0:
+            return HttpResponse("'id' must be greatest when zero")
```

Comment:
Just for future usage: here and in other places it is better to return error to a client in case of invalid input. E.g. `HttpResponseBadRequest`

Reply:
Ok, thanks!